packages/react-cosmos/src/server/embed-modules-webpack-loader.js

```diff
@@ -32,14 +32,11 @@ module.exports = async function embedModules(source: string) {
   const componentModuleCallls = convertPathsToRequireCalls(
     keys(deprecatedComponentModules).map(c => deprecatedComponentModules[c])
   );
+  const { componentsCommonDir, contextCall } = getContextCall(fixtureFiles);
```

Comment:
I hope you don't mind the nitpicking, but can you put `componentsCommonDir = commondir(paths)` here and then just return the `contextCall` from `getContextCall`, as the function name suggests. This way you also just need to send the context path to `getContextCall` instead of all the fixture files.

Reply:
I don't mind at all, that makes sense! Thought about it but didn't do anything. Thanks for the comment!